client/ayon_core/hosts/hiero/api/plugin.py

```diff
@@ -604,31 +607,42 @@ def load(self):
         return track_item
 
 
-class Creator(LegacyCreator):
+class HieroCreator(Creator):
     """Creator class wrapper
     """
     clip_color = "Purple"
     rename_index = None
 
-    def __init__(self, *args, **kwargs):
-        super(Creator, self).__init__(*args, **kwargs)
-        import ayon_core.hosts.hiero.api as phiero
-        self.presets = get_current_project_settings()[
-            "hiero"]["create"].get(self.__class__.__name__, {})
 
+    def apply_settings(self, project_settings):
+        resolve_create_settings = (
+            project_settings.get("hiero", {}).get("create")
+        )
+        self.presets = resolve_create_settings.get(
```

Comment:
Let's ensure that any code taken from other hosts is not committed. All namespaces in variables or other code parts should be related to Hiero.